Python/pythonrun.c

```diff
@@ -1236,12 +1236,6 @@ Py_CompileStringExFlags(const char *str, const char *filename_str, int start,
 }
 
 /* For use in Py_LIMITED_API */
```

Comment:
I think this comment should also go -- it looks like it only applies to the (now deleted) function.

Reply:
Oh, good point. I was confused by the diff and though that applied to the restored functions. FIxed in e27e218